src/openforms/authentication/tests/test_static_variables.py

```diff
@@ -0,0 +1,84 @@
+from django.test import TestCase
+
+from openforms.authentication.constants import AuthAttribute
+from openforms.authentication.tests.factories import AuthInfoFactory
+from openforms.forms.models import FormVariable
+
+
+class TestStaticVariables(TestCase):
+    def test_auth_static_data(self):
+        auth_info = AuthInfoFactory.create(
+            plugin="test-plugin",
+            attribute=AuthAttribute.bsn,
+            value="111222333",
+            machtigen={AuthAttribute.bsn: "123456789"},
+        )
+
+        static_data = FormVariable.get_static_data(auth_info.submission)
+
+        self.assertEqual(5, len(static_data))
+        self.assertEqual(
+            {
+                "plugin": "test-plugin",
+                "attribute": AuthAttribute.bsn,
+                "value": "111222333",
+                "machtigen": {AuthAttribute.bsn: "123456789"},
+            },
+            static_data[1].initial_value,
```

Comment:
I'd get rid of the magic numbers here by doing some destructuring:

```py
form_auth_var, auth_bsn_var, auth_kvk_var, auth_pseudo_var = static_data
```
and then do the assertions on those variables.